app/assets/stylesheets/application.css.scss.erb

```diff
@@ -381,3 +381,14 @@ td, th {
   text-align: center;
   padding: 10px;
 }
+
+input[type=text].modal-text {
+  height: 1.8rem;
+}
+
+.modal-radio[type="radio"]:not(:checked),
+.modal-radio[type="radio"]:checked {
```

Comment:
`checked` and `not(:checked)` are mutually exclusive. No need to specify the two... this could easily be `.modal-radio[type="radio"]`